### ISSUE TITLE
Extra fields to Blog Plugin

### ISSUE THREAD

**Issue by ayumihamsaki** [rainlab/blog-plugin#241]
Sorry just started using October cms sorry in advance if this question has been asked before or not?

Is it possible to update this plugin or copy it and edit the code or maybe add an extra plugin to it to be able to add extra features...?

Basically the blog plugin has a title and a description (pretty basic stuff)

I would like to add more fields like:

title
description
main image
article
alt title
keywords
publish date
modified date
creation date

etc...

Basically I need to create a lot more fields so I can link everything up to schema.org JSON-LD.

So what is the best method to do this please? Should I create a new plugin in builder, or copy the plugin and edit the code under a new name or create another plugin which can merge into this one or something else?

Thanks in advance.

**Comment by daftspunk** [comment]
[This screencast](https://vimeo.com/108040919) demonstrates the two primary ways you can extend another plugin. If it is as simple as adding some extra fields, it is a good idea to extend so you can retain the improvements moving forward. Otherwise if you determine that you need complete control, you are welcome to fork the plugin as it is open sourced.

**Comment by ayumihamsaki** [comment]
Thank you, I thought I watched all your videos but must of missed that one. Thank you for a quick response.
